assets/css/videojs-share.css

```diff
@@ -0,0 +1,7 @@
+/**
+ * videojs-share
+ * @version 3.2.1
+ * @copyright 2024 Mikhail Khazov <mkhazov.work@gmail.com>
+ * @license MIT
+ */
```

Comment:
Why are you adding `videojs-share` to the invidious repository, when it's already part of the VideoJS dependencies, and downloaded from NPM?

Reply:
Just testing stuff, hence why it's still in a draft.

I did find another alternative to videojs-share for videojs 8 here: https://www.npmjs.com/package/@viostream/videojs-share-menu but it's not ready out of the box.